tests/unit/test_charm.py

```diff
@@ -0,0 +1,348 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Unit tests for the bind module."""
+
+import json
+import logging
+from unittest.mock import patch
+
+import ops
+import pytest
+import scenario
+from scenario.context import _Event  # needed for custom events for now
+
+import tests.unit.helpers
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures("context")
+@pytest.mark.usefixtures("base_state")
+def test_start_wo_peer_relation(context, base_state):
```

Comment:
Small typo `wo` instead of `no`

Reply:
Yes, it was "without". I replaced all the `_wo_` with `_without_` and `_w_` with _with_`